portal-wire-protocol.md

```diff
@@ -10,6 +10,16 @@ Sub-protocol using the wire protocol **MAY** choose to exclude certain message t
 
 Unsupported messages **SHOULD** receive a `TALKRESP` message with an empty payload.
 
+## Protocol Version
+
+The portal wire protocol is versioned using whole number versins.  The current version is `1`.
```

Comment:
```suggestion
The portal wire protocol is versioned using unsigned integers.  The current version is `1`.
```

There was a spelling mistake, but I think this would be a bit more clear then saying `versioned using whole numbered versions` `unsigned integer` just feels a bit more clear, but idk up to you